README.md

```diff
@@ -184,20 +234,63 @@ To run tests and code coverage, use the following command:
 By default, a [Husky](https://github.com/typicode/husky) pre-commit hook is defined in `package.json`, which runs linting, testing, and code coverage before the commit is made. If either the linting, testing, or code coverage fails, the pre-commit hook prevents the changes from being committed. Review the results and fix the problems, or modify the thresholds to allow the commit to succeed.
 
 ### Experience tests
+
 You can verify the state of your locally running application by using the Selenium UI test script that is included in the `scripts` directory.
 
-### Debugging your local app
+## Enhancing your app with additional services
```

Comment:
This section covers adding services via CLI, but not via Console. Do we want to cover Console as well?

Reply:
yes we do...will work on this